src/platform/unittest/DataPathTest.cpp

```diff
@@ -49,6 +49,7 @@ struct QuicAddr
     }
 
     void Resolve(QUIC_ADDRESS_FAMILY af, const char* hostname) {
+        UNREFERENCED_PARAMETER(af);
```

Comment:
For all C++ code, if the variable is unused, wrap the variable in a comment. `QUIC_ADDRESS_FAMILY /* af */, ...`